weblate/machinery/tests.py

```diff
@@ -1023,6 +1042,62 @@ def test_cache(self):
         self.assertEqual(len(responses.calls), 0)
 
 
+@override_settings(MT_LIBRETRANSLATE_API_URL="URL")
+class LibreTranslateTranslationTest(BaseMachineTranslationTest):
+    MACHINE_CLS = LibreTranslateTranslation
+    EXPECTED_LEN = 1
+    ENGLISH = "EN"
+    SUPPORTED = "ES"
+    NOTSUPPORTED = "CS"
```

Comment:
These should be language codes as exposed by the service, so probably:

```suggestion
    ENGLISH = "en"
    SUPPORTED = "es"
    NOTSUPPORTED = "cs"
```